doc/Home.md

```diff
@@ -47,6 +47,7 @@ Friendica Documentation and Resources
 * [Protocol Documentation](help/Protocol)
 * [Database schema documantation](help/database)
 * [Class Autoloading](help/autoloader)
+* [Using Composer](help/Composer)
```

Comment:
Can you add this link to `doc/de/Home.md` as well and add a `((EN)` behind it as indicator for the text being available only in English at the moment.